helsingborg-stad/helsingborg-io-sls-api
# feat(viva-status-api): add api for viva-status

## Explain the changes you’ve made

Add new API endpoint for fetching VIVA status codes for a given person.

## Explain why these changes are made

Will be used to show VIVA status code information in the app.

## How to test

1. Checkout this branch
2. Deploy (deploy [resources #128 first](https://github.com/helsingborg-stad/helsingborg-io-sls-resources/pull/128)).
3. Call `/viva-status` (with valid `Authorization` header)
4. Verify codes are returned
